lib/interceptor.js

```diff
@@ -43,8 +43,12 @@ function Interceptor(scope, uri, method, requestBody, interceptorOptions) {
     this.optional = false;
 }
 
-Interceptor.prototype.optionally = function optionally() {
-    this.optional = true;
+Interceptor.prototype.optionally = function optionally(value) {
```

Comment:
We're stuck not using default argument values here, since we're not on ES6 :sob:

Reply:
yeah I know, lots of legacy and moving parts. Moving to ES6 will be the fun part, we have to get good code coverage and solid tests first. If you’d like to help out, we coordinate our efforts at #1077